sentry/src/main/java/io/sentry/ISerializer.java

```diff
@@ -12,6 +12,9 @@
 public interface ISerializer {
   <T> @Nullable T deserialize(@NotNull Reader reader, @NotNull Class<T> clazz);
 
+  <T> @Nullable T deserialize(
```

Comment:
@adinauer Since we don't do serialization inside the interceptor anymore, this change could be reverted. Or is this something we want to keep?

Reply:
I'd remove for now, as it seems easy to add back when needed.